src/components/fieldset/notes.md

```diff
@@ -0,0 +1,16 @@
+# Designer Notes
```

Comment:
Notes not appearing in story:

![image](https://user-images.githubusercontent.com/26925206/53957276-d5c24080-40d5-11e9-89f0-4c898d67d17b.png)